Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -19,7 +19,6 @@ file that was distributed with this source code.
     <select id="{{ id }}_autocomplete_input_v4" data-sonata-select2="false"
         {%- if read_only is defined and read_only %} readonly="readonly"{% endif -%}
         {%- if disabled %} disabled="disabled"{% endif -%}
-        {%- if required %} required="required"{% endif %}
```

Comment:
Isn't select2 supposed to remove it just before submit? Not sure why we are building the hidden fields ourselves either.